### ISSUE TITLE
Add uinput module to distro so we can use custom mouse mappings (ex: left mouse)

### ISSUE THREAD

**Issue by av-emogai** [microsoft/WSL#11270]
**Is your feature request related to a problem? Please describe.**
I would like to use a left mouse in the Ubuntu WSL Distribution
The inbuilt input-remapper in Ubuntu 22.04 does not work as the uinput module is not present in the out of box ubuntu for WSL2
 The service fails to start with the error: not able to see uinput file

**Describe the solution you'd like**
I would like to use a left mouse in the Ubuntu WSL Distribution; Other users on the web are looking for custom mouse buttons (4th and 5th button for gaming, etc)

**Describe alternatives you've considered**
Custom ubuntu distribution for 22.04 

**Additional context**
Any other way to get a left mouse to work inside Ubuntu for WSL? Please advice.

**Comment by sirredbeard** [comment]
I found a related question with a possible solution [here](https://answers.microsoft.com/en-us/surface/forum/all/left-handed-mouse-in-wsl2-ubuntu/0f004d7d-2477-459a-9b52-7f06124abbfc).

**Comment by symbioid** [comment]
Yes please.  I can't get remapping working with xmodmap, and systemd is wonky with tmux, so I'm using init.d...  I tried to install keyd for kernel level remapping, but because modules aren't allowed (or require a whole kernel compilation to support it) the only way to remap that I've found is MS Powertoys. But THAT only works if you have it in Admin mode, which is not something I feel comfortable doing.

If we could have uinput built in we could remap keys easily and natively.

I don't understand the logic of this, since so many vim users remap Caps-Lock.  The claim is "this is for devs, not a full linux distro) - but since devs remap it seems like something you'd want to have built in.

The "related solution" won't work (love those "MS Support" pages that never knows what they're actually dealing with). Blind leading blind.  xmodmap doesn't work, and the op even says (misspelled) they tried "xmode" (xmod).  I have spent a LOT of hours figuring a solution that works and doesnt' interfere with TMUX.

uinput compiled into the default kernel would allow these remaps (and probably even more/better features).  PLEASE MS, ADD THIS!
